### ISSUE TITLE
Cryptocurrency payments for services

### ISSUE THREAD

**Issue by rskikuli** [uradotdesign/task-management#9]
We are planning to accept payments through cryptocurrency. This means we need to:

- research about the best cryptocurrency(ies) that we will accept: @elioqoshi & Redon.
- creation of wallet(s): @Borisbudini 
- image promoting the initiative (needed for website blog post and social media): @AnXh3L0 
- website blog post & social media: @silvaarapi 
- integration at ura.design website: @Borisbudini

**Comment by elioqoshi** [comment]
Our Ledger Nano S wallet should ship this week and I will have it by middle of December. It accepts ETH, BTC and LTC so we should use that for storing cryptocurrency

**Comment by AnXh3L0** [comment]
Updating issue, Elio received the Ledger cryptowallet, it will be used for storing available funds.
About the cryptocurrencies that we should accept, I believe ETH, BTC would be the best for the moment.

**Comment by elioqoshi** [comment]
I also received some BTC and ETH to start off with (very little, less than 200 USD equivalent). Let's think of some way to incorporate this into our current website and how we would quote clients since cryptocurrency prices change constantly.

**Comment by kominoshja** [comment]
@elioqoshi where are we with this?

**Comment by elioqoshi** [comment]
Where are we supposed to be with this? Is there any concrete next step of action? Our ETH and BTC addresses are on Patreon and the only thing which I think could be done now is that we could add this information on our website. but that is dependent on the new website, which @AnXh3L0 leads.

**Comment by kominoshja** [comment]
Okay then, task postponed until the new website is ready!

**Comment by AnXh3L0** [comment]
I believe these tasks are now complete - shall we close this issue?

**Comment by rskikuli** [comment]
Blog post published, but we should also add the payment methods at one of our static pages, maybe here https://ura.design/about @AnXh3L0 . After this is done I'll close this.

**Comment by elioqoshi** [comment]
About is not the right place to put that. If anywhere, Services would be best to mention that we accept cryptocurrencies. However it doesn't make sense to show our addresses anywhere, unless we have a page to promote Identihub and donations for it.

**Comment by rskikuli** [comment]
OK. Closing this than since all the other tasks are done and creating a page for Identihub should be on another issue.

**Comment by elioqoshi** [comment]
No need to close this. The methods need to be added to the services page. Once that is done this should be closed.

**Comment by rskikuli** [comment]
Ok, when it is done please close it since I don't think I have an opinion on this.

**Comment by rskikuli** [comment]
There is a blog post about this https://ura.design/2018/03/31/introducing-cryptopayments-for-our-services. @AnXh3L0 please coordinate with @marianaballa to show how the update of the page can be done. 
Deadline for the implementation: 07.06.208.

**Comment by rskikuli** [comment]
@AnXh3L0 @marianaballa please add the methods of payment of the 'Services' page as requested by @elioqoshi . Ping me once it is done and I'll close the issue.

**Comment by AnXh3L0** [comment]
@rskikuli I have added that we accept cryptocurrency payments for our services in the services page and linked it to the blogpost for the cryptos.

**Comment by elioqoshi** [comment]
I think that's a bit out of place like that but we can iterate it later. Thanks
